src/Stepper/StepperHeader.jsx

```diff
@@ -30,11 +31,11 @@ function StepList({ steps, activeKey }) {
 
 const PageCount = ({ activeStepIndex, totalSteps }) => `Step ${activeStepIndex + 1} of ${totalSteps}`;
 
-function StepperHeader({ className, PageCountComponent }) {
+function StepperHeader({ className, PageCountComponent, compactWidth }) {
   const { steps, activeKey } = useContext(StepperContext);
   const windowDimensions = useWindowSize();
-  // assume about 200px per step
-  const isCompactView = windowDimensions.width < (steps.length * 200);
+  const breakpointWidth = breakpoints[compactWidth].maxWidth;
```

Comment:
I'm wondering if this might need to be more defensive, e.g. if consumer passes `compactWidth` that isn't supported or available as a property in `breakpoints`, I believe this would result in a JS error.

Is there a way to handle when `breakpoints[compactWidth]` is undefined?

<img width="707" alt="image" src="https://github.com/openedx/paragon/assets/2828721/2d3b025a-923c-4329-bca8-d15cd1523d51">

Reply:
Great catch! I've tried feeding two birds with one worm with an OR operator that will first check to see if the passed in shorthand value qualifies.